### DISCUSSION TITLE
Invalid destination when starting rip

### DISCUSSION THREAD

**Discussion by robertj352003** [jlesage/docker-handbrake#357] [Q&A] [answered]
I am getting this message after hitting the start button.
![Screenshot 2024-06-14 235234](https://github.com/jlesage/docker-handbrake/assets/166472031/8f517dcd-0020-4a96-bc0b-1987d4f0a0fc)

**Comment by jlesage** [comment, marked as answer]
Usually output files are saved under `/output`. If you want them to be under `/storage`, make sure to configure that associated volume mapping to be read/write.
